lmdeploy/messages.py

```diff
@@ -293,6 +293,11 @@ def __post_init__(self):
         assert self.device_type in [
             'cuda', 'ascend'
         ], (f'invalid device_type: {self.device_type}')
+        if self.quant_policy == 4:
+            import triton
+            from packaging import version
+            assert version.parse(triton.__version__) > version.parse('2.1.0'),\
+                'triton version should be >=2.2.0 for int4 quantization'
```

Comment:
2.2.0 failed with error:

```
loc(callsite(callsite("miniconda3/envs/torch2.0/lib/python3.9/site-packages/triton/language/standard.py":111:23 at "mdeploy/lmdeploy/pytorch/kernels/cuda/fill_kv_cache.py":29:38) at "lmdeploy/lmdeploy/pytorch/kernels/cuda/fill_kv_cache.py":297:53)): error: 'llvm.intr.minnum' op operand #0 must be floating point LLVM type or LLVM dialect-compatible vector of floating point LLVM type, but got 'i16'
Pass execution failedLLVM ERROR: Failed to translate TritonGPU to LLVM IR.
```

Reply:
It was raised by changing `tl.minimum(tl.min(val1, 1), tl.min(val2, 1))` to `tl.min(tl.minimum(val1, val2), 1)` since `val1` and `val2` are uint8 values which is not supported by minimum.